providers/oauth/oauth.httpobserver.js

```diff
@@ -67,7 +71,9 @@ FirefoxTabsAuth.prototype.launchAuthFlow = function (authUrl, stateObj, continua
 
   complete = function (location) {
     httpRequestObserver.unregister();
-    gBrowser.selectedTab = previousTab;
+    if (interactive) {
```

Comment:
Similar comment to freedom-for-chrome.
I don't believe this code matches either behavior that we want and/or the behavior of https://developer.chrome.com/apps/identity

It'll still open the tab, so I'm not sure what's the point of an interactive flag parameter